docs/include/plugin_header.asciidoc

```diff
@@ -1,43 +1,21 @@
-ifeval::["{versioned_docs}"!="true"]
 [subs="attributes"]
 ++++
 <titleabbrev>{plugin}</titleabbrev>
 ++++
-endif::[]
-ifeval::["{versioned_docs}"=="true"]
-[subs="attributes"]
-++++
-<titleabbrev>{version}</titleabbrev>
-++++
-endif::[]
 
 * Plugin version: {version}
 * Released on: {release_date}
 * {changelog_url}[Changelog]
 
-ifeval::["{versioned_docs}"!="true"]
-
 For other versions, see the
 {lsplugindocs}/{type}-{plugin}-index.html[Versioned plugin docs].
 
-endif::[]
-
-ifeval::["{versioned_docs}"=="true"]
-
-For other versions, see the <<{type}-{plugin}-index,overview list>>.
-
-To learn more about Logstash, see the {logstash-ref}/index.html[Logstash Reference].
-
-endif::[]
-
 ifeval::["{default_plugin}"=="0"]
-ifeval::["{versioned_docs}"!="true"]
 
 ==== Installation
 
 For plugins not bundled by default, it is easy to install by running +bin/logstash-plugin install logstash-{type}-{plugin}+. See {logstash-ref}/working-with-plugins.html[Working with plugins] for more details.
 
-endif::[]
 endif::[]
```

Comment:
This cond'l statement left in on purpose.  It pulls in Installation instructions for plugins not installed by default.